java/dataflow-pardo-helloworld/README.md

```diff
@@ -32,6 +32,14 @@ pipeline that writes the words "Hello" and "World" into Cloud Bigtable.
 
 `$ mvn compile exec:java -Dexec.mainClass="com.example.bigtable.dataflow.pardo.HelloWorldBigtablePardo" -Dexec.args="--runner=BlockingDataflowPipelineRunner --project=<PROJECTID> --bigtableProjectId=<PROJECTID> --bigtableClusterId=<CLUSTERID> --bigtableZoneId=<ZONE> --bigtableTableId=Dataflow_test --stagingLocation=gs://<STORAGE_BUCKET>/staging"`
 
+Note - Occasionally, you see something like -- it is a benign issue and should be fixed shortly:
```

Comment:
Note: Occasionally, you may see log messages similar to the following. It is safe to ignore these warnings, which will be fixed in a future release: